CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file. For change
 ## Unreleased
 - updated contrib docs to include instructions to update tests for new releases
 
+## 2.0.2 2018-01-24
+- Drop node 4.3.2 support.
```

Comment:
This will break anyone running lambda-cfn v2.x on node 4.3.2 right? So we should bump this to 3.x to prevent that?

Reply:
@zmully, not it will not affect any running lambda function. Because every lambda runs in its own stack.

Reply:
@zmully, You are right. It's better a bump a 3.x version. I will make that modification.